### ISSUE TITLE
Can't connect to torrent client qBitTorrent v3.3.14

### ISSUE THREAD

**Issue by kandovit** [SchizoDuckie/DuckieTV#928]
**What build of DuckieTV are you using (Standlone / Chrome Extension (New Tab / Browser Action))**
Using Chrome Extension - New Tab

**What version of DuckieTV are you using (Stable 1.1.x / Nightly yyyymmddHHMM)**
Stable v1.1.4

**What is your Operating System (Windows, Mac, Linux, Android)**
Windows 10 Pro (64x) 10.0.15063 

**Describe the problem you are having and steps to reproduce if available**
Can't connect to torrent client using WEB UI
I've defined under Settings --> Torrent --> My torrent client as "QBITTORENT 3.2+"
Then went to "QBITTORRENT 3.2+ INTEGRATION" - Defined there:
Address: http://localhost
Port: 8080
Use authentication: V
Username: admin
Password: 123456

Also defined those settings under the torrent client as mentioned at the set-up instructions.

Status = Not Connected.

I've checked under F12 (Dev tools) -- Console
I got error 401, Unauthorized
So I've:
1. Opened localhost:8080 on my chrome browser and logged in using admin:123456 -- Connected with no problem
2. Closed all and reopened -- Didn't help
3. Went again to F12, and saw that it's defined correctly, saw admin:123456..
4. Then I've clicked on the link under the console line that failed, and it redirected my to localhost:8080/login --> Got "Fails."
5. So I've went to localhost:8080 again and logged in, then went to localhost:8080\login and got "Ok."
6. Tried to refresh the extension, still got the same error... 

Question: Does it should redirect to "http://localhost:8080/login" or only to "localhost:8080"?

**Attach any DuckieTV statistics or Developer Console logs if available**
Don't know how to add it.. 

Thank you!

**Comment by garfield69** [comment]
Duplicate of #903.      

qBitTorrent *3.3.13* introduced Cross-Site Request Forgery protection for their WEBUI API. However, they set the rules too tight, and ended up denying any WEBUI API user except their own internal one.     
Issue qbittorrent/qBittorrent#6882 was raised, and *3.3.14* loosened the rules enough to allow other clients access, but not without additional supporting changes to DuckieTV's own qBt 3.2+ client.     
Commit 7a2bbb9a4ec123f6928c805599e2f69fe352375f resolves the problem.     

So at this point you have several choices:      
- Wait for [DuckieTV 1.1.5](https://github.com/SchizoDuckie/DuckieTV/releases) to be published, I don't have an ETA at this stage, it's ready, waiting on Schizoduckie.        
- Grab a [DuckieTV Nightly](https://github.com/DuckieTV/Nightlies/releases) so you can connect to qBt *3.3.14* now      
    [edit] wiki should you need it [installing DuckieTV Nightly as a Chrome Browser Extension](https://github.com/SchizoDuckie/DuckieTV/wiki/How-to-Install-a-Development-version-of-the-DuckieTV-Chrome-Browser-Extension-NIGHTLY)
- Downgrade your [qBitTorrent to 3.3.12](https://sourceforge.net/projects/qbittorrent/files/qbittorrent-win32/qbittorrent-3.3.12/qbittorrent_3.3.12_x64_setup.exe/download) so you can connect with DuckieTV *1.1.4* .
